sysfs_linux.go

```diff
@@ -0,0 +1,70 @@
+package mlxdevm
+
+// The following operations are currently not supported by the 'mlxdevm' devlink:
+// - Retrieving the auxiliary device name of an SF port, e.g., mlx5_core.sf.3.
+// - Unbinding the SF port from the default configuration driver and binding it to the actual SF driver.
+// These operations must be performed through sysfs. While we are looking for
+// adding devlink support for them in the long term, these functions within this
+// file can be used to accomplish the task until that support is implemented.
+
```

Comment:
This is unrelated to mlxdevm-go lib it does not deal with sysfs.

Reply:
Unbind/bind SF is a mandatory step to deploy SF,  does mlxdevm generic netlink family support it now?

The reason of adding these functions is every user of this lib need to write similar code to deploy SF port. If they shouldn't be part of mlxdevm-go lib, do you have some suggestion where should they go? 

CC @venuiyer

Reply:
@adrianchiris understand your concern. I think we need to determine what is expected from the caller 
when using some fns from this lib. E.g. when the trust function is called (the other PR), the SF should
not be attached, should we do that part of the trust call itself or expect the caller to make sure it is
detached before calling the trust?

It'll be nice to have the bind/unbind part of some lib, else every user will need to the same since 
as Lei says binding/unbinding is part of the seq to use the SF.
https://docs.nvidia.com/doca/sdk/nvidia+bluefield+dpu+scalable+function+user+guide/index.html (step 3)

Admittedly, it is not via mlxdevm; so should we instead have his part in sriovnet lib?